scico/linop/_misc.py.BAK

```diff
@@ -0,0 +1,219 @@
+# Copyright (C) 2020-2022 by SCICO Developers
```

Comment:
This file meant to be here?

Reply:
No! Good catch, thanks. Not sure how that sneaked in there.